modules/SofaGraphComponent/SofaGraphComponent_test/SceneChecker_test.cpp

```diff
@@ -60,18 +63,71 @@ struct SceneChecker_test : public Sofa_test<>
         ASSERT_NE(root.get(), nullptr) ;
         root->init(ExecParams::defaultInstance()) ;
 
+        SceneCheckerVisitor checker(ExecParams::defaultInstance());
+        checker.addCheck( SceneCheckMissingRequiredPlugin::newSPtr() );
+
         if(missing)
         {
             EXPECT_MSG_EMIT(Warning);
-            SceneCheckerVisitor checker(ExecParams::defaultInstance());
             checker.validate(root.get()) ;
         }else{
             EXPECT_MSG_NOEMIT(Warning);
-            SceneCheckerVisitor checker(ExecParams::defaultInstance());
             checker.validate(root.get()) ;
         }
     }
 
+    void checkDuplicatedNames()
+    {
+        EXPECT_MSG_EMIT(Error) ;
```

Comment:
@damienmarchal, I think you meant `EXPECT_MSG_NOEMIT` here.

Reply:
Correct #405

Reply:
Haha didn't see that yet, will do ;-)

Reply:
I was so ashamed to have merged something with obviously broken test that I quickly fix it ;)